src/extension.ts

```diff
@@ -51,6 +54,27 @@ async function init(context: vscode.ExtensionContext, git: GitApiImpl, credentia
 		}
 	});
 
+	context.subscriptions.push(git.onDidPublish(async e => {
+		// Only notify on branch publish events
+		if (!e.branch || PersistentState.fetch(PROMPTS_SCOPE, PROMPT_TO_CREATE_PR_ON_PUBLISH_KEY) === false) {
+			return;
+		}
+
+		const reviewManager = reviewManagers.find(reviewManager => reviewManager.repository.rootUri.toString() === e.repository.rootUri.toString());
```

Comment:
there are a couple of tslint warnings here, `reviewManager` is a shadowed variable and some missing semicolons below